client/src/App.test.js

```diff
@@ -1,9 +1,7 @@
-import React from "react";
-import { render, screen } from "@testing-library/react";
-import App from "./App";
+function sum(a, b) {
+  return a + b;
+}
 
-test("renders learn react link", () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+test("adds 1 + 2 to equal 3", () => {
+  expect(sum(1, 2)).toBe(3);
```

Comment:
This test doesn't make much sense here, especially if the file is named `App.test.js`.
You can just test if the App renders:

```js
test("renders App", () => {
  render(<App />);
});
```